graph-db/src/utils/cmr/harvestCmrCollections.js

```diff
@@ -0,0 +1,25 @@
+const { fetchPageFromCMR } = require('./fetchPageFromCmr');
+const { clearScrollSession } = require('./clearScrollSession');
+
+exports.harvestCmrCollections = async () => {
+    let { scrollId, response } = await fetchPageFromCMR();
+    let partitionedSearchResults = [];
+    let continueScroll = true;
+
+    partitionedSearchResults.push(response);
+    while (continueScroll){
```

Comment:
whitespace nitpick, space between `){`

Reply:
Would you also ever want to stop at a pre-defined size instead of getting everything?

Reply:
I don't see that needing to happen for this use case